assets/src/conversion/index.js

```diff
@@ -104,130 +114,115 @@ class Conversion extends Component {
+							<TextControl
+								label={ __( 'Conversion batch size (entries per batch)' ) }
+								disabled={ true }
+								value={ conversionBatchSize }
+							/>
+							<hr />
+							<h2>{ __( 'Reset ongoing conversion' ) }</h2>
 							<p>
 								{ __(
-									"Looks like some posts weren't converted properly. This could have happened due to unexpected/unsupported Post content (valid HTML source is expected as Post content), or due to technical reasons (e.g. your server experiencing difficulties). You may retry converting these posts now."
+									'In case that your active conversion browser tab has been closed by accident, or it has been interrupted and closed unexpectedly, you may reset the conversion status here, and start converting all over again.'
 								) }
 							</p>
-							<ul>
-								<li>
-									{ __( 'number of posts which failed getting converted:' ) }{' '}
-									<b>
-										<u>{ countFailedConverting }</u>
-									</b>
-								</li>
-							</ul>
-
-							<input
-								type="submit"
-								className="large"
-								value={ __( 'Retry Failed' ) }
-								onClick={ this.handleOnClickInitializeRetryFailed }
+							<Notice noticeText={ __( 'This will enable you to restart the conversion, but any previous results may be lost.' ) } isWarning />
+							<Card noBackground className="newspack-card__buttons-card">
+								<Button
+									isPrimary
+									onClick={ this.handleOnClickResetConversion }
+								>
+									{ __( 'Reset conversion' ) }
+								</Button>
+							</Card>
+						</Card>
+					</Grid>
+				</Fragment>
+			);
+		} else {
+			return (
+				<Fragment>
+					<div className="newspack-logo-wrapper">
+						<NewspackLogo />
+					</div>
+					<Grid>
+						<FormattedHeader
+							headerIcon={ <UnarchiveIcon /> }
+							headerText={ __( 'Run conversion' ) }
+							subHeaderText={ __( 'The queued content will be converted to Gutenberg Blocks.' ) }
+						/>
+						<Card>
+							{ !! someConversionsFailed && (
+								<Fragment>
+									<Notice noticeText={ __( 'Looks like some entries weren\'t converted properly. You may retry converting these.' ) } isError />
+									<TextControl
+										label={ __( 'Number of failed entries' ) }
+										disabled={ true }
+										value={ countFailedConverting }
+									/>
+									<Card noBackground className="newspack-card__buttons-card">
+										<Button
+											isPrimary
+											onClick={ this.handleOnClickInitializeRetryFailed }
+										>
+											{ __( 'Retry conversion' ) }
+										</Button>
+									</Card>
+									<hr />
+								</Fragment>
+							) }
+							<Notice noticeText={ __( 'This page will automatically reload for every batch.' ) } isPrimary />
+							<TextControl
+								label={ __( 'Number of entries to be converted' ) }
+								disabled={ true }
+								value={ queuedEntries }
+							/>
+							<TextControl
+								label={ __( 'Total conversion batches queued' ) }
```

Comment:
Proposing change in text:
Total conversion batches

Reply:
Fixed in ec06523